tests/libcxx/expected_results.txt

```diff
@@ -53,7 +53,8 @@ std/utilities/memory/default.allocator/allocator.ctor.pass.cpp FAIL
 # LWG-3197 "std::prev should not require BidirectionalIterator" (New)
 std/iterators/iterator.primitives/iterator.operations/prev.pass.cpp FAIL
 
-# Itanium ABI assumptions that current_exception and rethrow_exception don't copy the exception object
+# Itanium ABI assumptions that current_exception and rethrow_exception don't copy the exception object.
+# The SKIPPED tests contain `XFAIL: msvc`, which is not well understood by the test harness.

```

Comment:
No change requested: I would have said "the MSVC-internal test harness" here because the GitHub test harness does understand `XFAIL: msvc`, but it's not worth resetting testing, this comment is a strict improvement.

Reply:
These tests successfully compile but fail to run. So they are affected by the ARM and ARM64 issue that you explained in

https://github.com/microsoft/STL/blob/a8888806c6960f1687590ffd4244794c753aa819/tests/libcxx/expected_results.txt#L1346-L1355